retirement_api/utils/ss_calculator.py

```diff
@@ -234,16 +293,23 @@ def get_retire_data(params, language):
                'note': '',
                'past_fra': False,
                }
-    BENS = results['data']['benefits']
-    current_age = get_current_age(dobstring)
     results['current_age'] = current_age
+    fra_tuple = get_retirement_age(yobstring)  # returns tuple: (year, momths)
+    if fra_tuple[1]:
+        FRA = "%s and %s months" % (fra_tuple[0], fra_tuple[1])
+    else:
+        FRA = "%s" % fra_tuple[0]
```

Comment:
Teensy weensy niggle: could we use `"{}".format()` string formatting for future-friendliness?